main.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'app'
+
+def display_main_menu
+  puts "\nMain Menu: "
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Rent a book'
```

Comment:
- [ ] When option 5 is selected to rent a book, you are first required to ask the user to select a book based on the number, thereafter he is to select a person based on the number and not id. please klindly go over the [reference video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) and follow the steps as shown there. :+1: :sparkles: 
![op](https://github.com/AliRezaBmeDu/school-library/assets/89055502/df03554d-43c0-48b4-abb0-3b447f55e54d)

Reply:
done